### PR TITLE
Ae scripts arguments

### PR DESCRIPTION
**Pull request by paetling**
review @jsingle

### CODE REVIEW

File: dusty/commands/bundles.py [hunk 1]
```diff
@@ -14,6 +14,9 @@ def list_bundles():
         table.add_row([bundle,
                        bundle_spec['description'],
                        u"✓" if bundle in activated_bundles else ""])
+    import logging
```
Comment (jsingle) [inline dusty/commands/bundles.py]: rogue import/debug code

File: dusty/commands/script.py [hunk 2]
```diff
@@ -1,5 +1,6 @@
-import textwrap
+# coding=utf-8
```
Comment (jsingle) [inline dusty/commands/script.py]: What's this?

**Comment by jsingle** [comment]
Couple minor things but this looks good.  Did you consider always passing any additional arguments to the script, rather than requiring a key in the specs to be set?

File: dusty/commands/script.py (hunk 2 above)
Comment (paetling) [inline dusty/commands/script.py]: this encodes the python file in unicode, which allows the check mark to be handled properly

**Comment by paetling** [comment]
I did consider it.  But i figure if you are adding a script to your .yml, you know at that time if it should take arguemnts.  If it should not, you really do not want people passing arguments to it.

**Pull request merged**